udn-density-l3-pods.go

```diff
@@ -65,7 +57,5 @@ func NewUDNDensityL3Pods(wh *workloads.WorkloadHelper) *cobra.Command {
 	cmd.Flags().StringVar(&churnDeletionStrategy, "churn-deletion-strategy", "default", "Churn deletion strategy to use")
 	cmd.Flags().IntVar(&iterations, "iterations", 0, "Iterations")
 	cmd.Flags().DurationVar(&podReadyThreshold, "pod-ready-threshold", 1*time.Minute, "Pod ready timeout threshold")
-	cmd.Flags().IntVar(&podsPerNode, "pods-per-node", 245, "Pods per node")
-	cmd.Flags().BoolVar(&svcLatency, "service-latency", false, "Enable service latency measurement")
```

Comment:
I do not think udn-l3 workload supports churning (yet) - can we disable it?
`cmd.Flags().BoolVar(&churn, "churn", false, "disable churning")`

Reply:
It looks like it does: https://github.com/kube-burner/kube-burner-ocp/blob/main/cmd/config/udn-density-l3-pods/udn-density-l3-pods.yml#L44-L49

Reply:
@venkataanil Can you confirm if you will be using Churning?

Reply:
> It looks like it does: https://github.com/kube-burner/kube-burner-ocp/blob/main/cmd/config/udn-density-l3-pods/udn-density-l3-pods.yml#L44-L49

IIUC - we might not have immediate/no value with churning, but if it's otherwise I'm ok to leave it as-is.